lib/jsdom/living/events/UIEvent-impl.js

```diff
@@ -22,10 +22,10 @@ function isWindow(val) {
 
 class UIEventImpl extends EventImpl {
   constructor(args, privateData) {
-    const eventInitDict = args[1] || UIEventInit.convert(undefined);
+    const eventInitDict = args[1];
 
     // undefined check included so that we can omit the property in internal usage.
-    if (eventInitDict.view !== null && eventInitDict.view !== undefined) {
+    if (eventInitDict && eventInitDict.view !== null && eventInitDict.view !== undefined) {
```

Comment:
When could `eventInitDict` be falsy? I would have though the Web IDL layer would have guaranteed it to be an object by now.

Reply:
When we get lazy and call it like `SomeEvent.createImpl(['name'])`